### ISSUE TITLE
Undefined message on non-JSON RabbitMQ subscribe

### ISSUE THREAD

**Issue by Alterock13** [golevelup/nestjs#276]
Hello,

I am discovering your project, trying to setup a nest microservice using more specific RabbitMQ features (namely, the usage of exchanges)

The issue is that my program should be able to parse non JSON incoming message from an exchange so I can process it and redirect it to another exchange.
While fiddling with the code, I discovered the option `allowNonJsonMessages` which seems to provide what I'm looking for.

However, even though I can receive JSON messages, non-JSON are just read as `undefined`.

Here is my code :

```js
import { RabbitSubscribe } from '@golevelup/nestjs-rabbitmq';
import { Injectable } from '@nestjs/common';
import { env } from 'src/env';
import { ConsumeMessage } from 'amqplib';

@Injectable()
export class MainService {
  @RabbitSubscribe({
    exchange: env.rabbit.exchange.consume,
    allowNonJsonMessages: true,
    routingKey: '',
    queue: env.rabbit.exchange.consume,
  })
  public async subHandler(message, rawMessage: ConsumeMessage) {
    console.log(rawMessage);
    console.log(message);
  }
}
```

Could you please enlighten me on what I might be doing wrong ?

Thanks in advance for your help.

**Comment by WonderPanda** [comment]
@Alterock13 What gets printed when you log the `rawMessage`? Currently receiving undefined to the first message is expected (although probably not the best design choice) but if you're not using JSON with the current version of the library it is up to you to handle the raw message based on what makes sense for your app

**Comment by Alterock13** [comment]
@WonderPanda Thanks for your reply. When I tried yesterday, `rawMessage` container the `fields`, `properties` and `content` but their own properties were `undefined`. After receiving your reply, I launched my rabbit cluster again and now I have content in `content`. I don't know what could have caused this on Rabbit side as the code I posted before hasn't changed.. Anyway, now it seems to work, sorry for bothering.

I have another issue/question about sending messages this time, but I'll post it in another thread.
Thanks again for your reply and this library.
